definition/data/sheets/FixedLists.json

```diff
@@ -196,5 +196,9 @@
   {"ID": "RpaType", "ListElementCode": "MORE_TIME", "ListElement": "More Time"},
   {"ID": "RpaType", "ListElementCode": "CCJ", "ListElement": "CCJ"},
   {"ID": "RpaType", "ListElementCode": "DEFENDANT_RESPONSE", "ListElement": "Defendant Response"},
-  {"ID": "RpaType", "ListElementCode": "PAID_IN_FULL", "ListElement": "Paid in Full"}
+  {"ID": "RpaType", "ListElementCode": "PAID_IN_FULL", "ListElement": "Paid in Full"},
+  {"ID": "ProceedOnPaperReason", "ListElementCode": "APPLICATION_BY_CLAIMANT", "ListElement": "Application by Claimant"},
```

Comment:
Can you please add a display order as 1, 2,3

Reply:
Done @SanchitaMoj